templates/docs/patterns/buttons.md

```diff
@@ -64,6 +64,14 @@ Should you wish to place a button after a line of inline text, as a CTA for exam
 View example of the inline button pattern
 </a></div>
 
+### Active
```

Comment:
Could we move it to be the last variant on the docs page, I think the following (dense, small, icon) are more common and could be earlier on the page.

`is-active` is quite an edge case, so it could be mentioned last.

Reply:
:+1: updated